src/wobjectdefs.h

```diff
@@ -33,6 +33,7 @@
 namespace w_internal {
 using std::index_sequence;  // From C++14, make sure to enable the C++14 option in your compiler
 
+#ifdef W_USE_CUSTOM_MAKE_INDEX_SEQUENCE
```

Comment:
Did you mesure the time of compiling the tests before and after?
I remember back when i did that, std::make_index_sequence would not even compile.
I don't want to drop support for old compilers.

Reply:
I have not done a survey of different compilers.
On my Laptop and Visual Studio 2019 the compile times for all commits decreased.

I added the switch to allow older compilers to stay fast.